clc/cmd/exec_context.go

```diff
@@ -107,6 +107,10 @@ func (ec *ExecContext) Props() plug.ReadOnlyProperties {
 	return ec.props
 }
 
+func (ec *ExecContext) ConfigPath() string {
```

Comment:
Does this have to be in `ExecContext`?

Reply:
I could not find anywhere else to put it since it depends on the Config Provider. Do you have a suggestion where I could put it?

Reply:
OK, let's keep it for now.
We may need to add a method that returns the config provider in the future.